packages/devtools/lib/src/debugger/debugger.css

```diff
@@ -42,6 +48,29 @@
 
 .debugger-items-list {
     overflow-y: scroll;
+    cursor: pointer;
+}
+
+.open-popup {
+    z-index: 1000;
+    position: absolute;
+    top: 0px;
+    left: 5px;
+    height: 200px;
+    background-color: white;
+    list-style: none;
+    border: 1px solid #d1d5da;
+    padding-left: 4px;
+    margin-vottrom: 2px;
+    cursor: pointer;
+    display: none;
+}
+
+.popup-textfield {
+    width: calc(100% - 105px);
```

Comment:
this sort of calc could also be nice for the height of the popup.

Reply:
Yep should be:

    height: calc(100% - 70%);